platform/solr/platform-solr-server-standalone/dependency-check-maven-config.xml

```diff
@@ -16,7 +16,16 @@
    ]]></notes>
         <cve>CVE-2016-3720</cve>
     </suppress>
-
+    <suppress>
```

Comment:
❓ Does the dependency-check-maven-config.xml suppression not cover this module?

Reply:
I don't believe it does; @clockard might know the answer better than I

Reply:
This module supplies its own so we generally need to put a copy of the suppression here as well. Just looked into this a little more and I am wondering if this is going to be a separate issue. We are shading in solr-core in solr-dependencies which include jackson 2.5.4 and 1.9.13

Reply:
Looks like we don't actually shade in databind only core and dataformat-smile so this shouldn't be an issue. I think that cve applies to all versions up through 2.8.10